src/components/PendingSell.js

```diff
@@ -101,7 +102,12 @@ class PendingSellUnstyled extends Component<Props, State> {
     let edgeTransaction
     await window.edgeProvider.chooseCurrencyWallet([info.currencyCode])
     try {
-      edgeTransaction = await window.edgeProvider.requestSpend([info])
+      const metadata: EdgeMetadata = {
+        name: 'Simplex',
+        category: 'Exchange:Sell Bitcoin',
```

Comment:
Change 'Sell Bitcoin' for Sell ${currencyCode}

Reply:
Ok.